libs/parallelism/algorithms/include/hpx/parallel/datapar/loop.hpp

```diff
@@ -252,7 +252,7 @@ namespace hpx { namespace parallel { namespace util {
                 static std::size_t constexpr size =
                     traits::vector_pack_size<V>::value;
 
-                for (std::int64_t lenV = std::int64_t(count - (size + 1));
+                for (std::int64_t lenV = std::int64_t(len - (size + 1));
```

Comment:
Please adher to our naming conventions (see here: https://github.com/STEllAR-GROUP/hpx/wiki/HPX-Source-Code-Structure-and-Coding-Standards)